tests/test_gaap.py

```diff
@@ -237,6 +237,13 @@ def testFlags(self, sigmas=[2.5, 3.0, 4.0], scalingFactors=[1.15, 1.25, 1.4]):
             The list of sigmas (in pixels) to construct the `GaapFluxConfig`.
         scalingFactors : `list` [`float`], optional
             The list of scaling factors to construct the `GaapFluxConfig`.
+
+        Notes
+        -----
+        This test will fail if config parameters that cause a measurement
+        failure is not present. Since the seeing in the test dataset is
```

Comment:
is not present -> are not present

Reply:
I've rewritten this part and the grammar error just magically vanished.